style.css

```diff
@@ -478,17 +499,275 @@ textarea {
+    padding-right: 14%;
+  }
+
+  .intro > h1 {
+    font-size: 45px;
+    font-family: Poppins-extrabold, sans-serif;
+    margin-top: 0;
+    margin-bottom: 0.32em;
+    padding-right: 40%;
+  }
+
+  .intro > p {
+    text-decoration: solid;
+    font-size: 16px;
+  }
+
+  .imgalin {
+    object-fit: contain;
+    max-width: 100%;
+    height: 100%;
+    margin: 0;
+  }
+
+  .section1 {
+    display: flex;
+    flex-direction: column;
+  }
+
+  header {
+    display: flex;
+    font-weight: 700;
+    font-family: Poppins-semibold, sans-serif;
+    font-size: 1.125em;
+
+    /* Primary/B400 */
+    color: #6070ff;
+    align-items: center;
+    justify-content: space-between;
+    background-position: bottom 0;
+    padding-left: 5%;
+    padding-right: 5%;
+    padding-top: 1%;
+    background-color: #fff;
+  }
+
+  .card-one,
+  .card-two,
+  .card-three,
+  .card-four,
+  .card-five {
+    display: grid;
+    grid-template-columns: 50% 50%;
+    justify-content: center;
+    padding: 5%;
+    height: auto;
+    width: auto;
+    gap: 5%;
+  }
+
+  .switch {
+    order: 1;
+  }
+
+  h2 {
+    font-size: 2.5em;
+    line-height: 1em;
+    font-family: Poppins-extrabold, sans-serif;
+    font-weight: 700px;
+    margin-bottom: 2%;
+  }
+
+  .section1 > div {
+    margin-top: 4%;
+    padding-bottom: 2%;
+    padding-top: 2%;
+  }
+
+  .grid-element {
+    margin-top: 10%;
+  }
+
+  .about {
+    display: flex;
+    flex-direction: column;
+    padding-left: 8%;
+    padding-right: 8%;
+    padding-top: 5%;
+    padding-bottom: 10%;
+    background-color: white;
+    margin-top: 5%;
+    border-top-right-radius: 19% 9%;
+  }
+
+  .skills {
+    display: grid;
+    grid-template-columns: auto auto auto;
+    width: 100%;
+    margin-top: 0.1rem;
+    gap: 3%;
+  }
+
+  .skills > ul {
+    display: flex;
+    flex-direction: column;
+    align-items: start;
+    margin-top: 3%;
+    font-weight: bolder;
+  }
+
+  .skills > ul > li {
+    margin-top: 5%;
+  }
+
+  .languages {
+    display: flex;
+    width: 100%;
+    align-items: first baseline;
+    height: 4rem;
+    justify-content: space-between;
+    margin-bottom: 5%;
+    margin-top: 0%;
+    flex-direction: row;
+  }
+
+  .language-DK {
+    display: flex;
+    align-items: center;
+    height: 4rem;
+    margin-top: 0.1rem;
+    width: 100%;
+    justify-content: space-between;
+  }
+
+  .language-skill-DK {
+    display: flex;
+    flex-direction: row;
+    width: 100%;
+    height: 115px;
+    background: #f7f7f9;
+    border-radius: 9px;
+    margin-top: 3%;
+  }
+
+  .contact {
+    display: flex;
+    flex-direction: column;
+    background-image: url("./images/contact-form-background-shapes.svg");
+    background-repeat: no-repeat;
+    margin-top: 1%;
+    padding-top: 15%;
+    padding-left: 1%;
+    border-radius: 0 0 0 0;
+    background-color: #6070ff;
+    background-size: 100%;
+    background-position: 85% 20%;
+    align-items: center;
+    padding-bottom: 40%;
+  }
+
```

Comment:
- [ ] Good job so far 👍👍 However, as suggested by the reviewer [here](https://github.com/Katended/mobilefirstproject2/pull/4#discussion_r1159713932), please make sure to remove the `border-radius` from desktop version of the portfolio as it is not present in figma template (desktop version) 🛠
- [ ] Also, please make sure to remove the excessive spaces to the bottom of the page as suggested by the previous reviewer ✨


Your design 🔽 | Figma guidelines 🔽
---| ---|
![image](https://user-images.githubusercontent.com/106119105/230496929-a6a17f61-9cdb-4335-8a02-c7822e382291.png) | ![image](https://user-images.githubusercontent.com/106119105/230496997-d5c27686-c951-40f8-9e23-f280be9cfff5.png)

Reply:
> * border-

@KanzaTahreem Please note that to remove border radius I hard to `border-radius: 0px;`

Reply:
@Katended, I believe the border radius she was talking about is the one on the inputs and the textarea.